src/main/java/core/basesyntax/Application.java

```diff
@@ -3,5 +3,10 @@
 public class Application {
     public static void main(String[] args) {
         // create three balls using class Lottery and print information about them in console
+        Ball[] balls = new Ball[3];
+        for (Ball ball : balls) {
+            ball = new Lottery().getRandomBall();
+            System.out.println(ball.toString());
+        }
```

Comment:
toString is called under the hood automatically
try to rewrite this without using array and creating only one lottery instance, not 3